src/src/openthread-core-efr32-config.h

```diff
@@ -47,19 +47,20 @@
 #endif
 
 #include "board_config.h"
+#include "em_device.h"
```

Comment:
It looks like this included file on [GSDK v4.1.2](https://stash.silabs.com/projects/GSDK/repos/gsdk/browse/protocol/openthread/platform-abstraction/efr32/openthread-core-efr32-config.h?at=v4.1.2#50) is `"em_msc.h"`. Does including `em_device.h` accomplish the same thing, and/or was it an intentional replacement?

Reply:
Yes, this is intentional. `em_msc.h` is the header file for our `Memory System Controller`. We don't need access to the definitions and types in `em_msc.h` across the entire openthread project, just in [flash.c](https://github.com/openthread/ot-efr32/blob/main/src/src/flash.c#L75-L92).

What we do need access to is the device-specific defines, etc provided by `em_device.h`, which `em_msc.h` does include, but including `em_device.h` directly is more correct.